charts/openebs/values.yaml

```diff
@@ -263,9 +263,17 @@ policies:
   monitoring:
     enabled: true
     image: "openebs/m-exporter"
-    imageTag: "2.6.0"
+    imageTag: "2.7.0"
 
 analytics:
   enabled: true
   # Specify in hours the duration after which a ping event needs to be sent.
   pingInterval: "24h"
+
+
+# To cleanup the validatingwebhook just before deletion of the charts
+# via a pre-delete webhook job
+hyperkubeImage:
+  repository: "k8s.gcr.io/hyperkube"
+  tag: "v1.12.1"
+  pullPolicy: "IfNotPresent"
```

Comment:
q_: please check if the tolerations can be added to the job?

Reply:
@prateekpandey14 @shubham14bajpai Can you check this. https://github.com/openebs/maya/blob/master/pkg/install/v1alpha1/jiva_volume.go#L1363

Reply:
We should probably reuse the tolertions set on m-apiserver instead of having users repeat the same thing again.

Reply:
```yaml
{{- if .Values.webhook.tolerations }}
      tolerations:
{{ toYaml .Values.webhook.tolerations | indent 8 }}
{{- end }}


```